faq/tests/settings/19.py

```diff
@@ -0,0 +1,107 @@
+import os
+from os.path import expanduser
+
+
+USE_TZ = True
+
+TIME_ZONE = "Africa/Johannesburg"
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.postgresql_psycopg2",
+        "NAME": "jmbo",
+        "USER": "postgres",
+        "PASSWORD": "",
+        "HOST": "",
+        "PORT": "",
+    }
+}
+
+xDATABASES = {
```

Comment:
I am not sure, I kept it since I tried to stay as close as possible to jmbo-post. I will ask Hedley about it on Monday when he is back from leave